bin/prep-release.php

```diff
@@ -62,7 +62,7 @@ function get_files() {
         "scripts/centos/ldap/users",
         "scripts/presets/*.{json}",
         "scripts/ssl/*.pem",
-        "scripts/*.{php,sh}",
+        "scripts/*.{json,php,py,sh}",
```

Comment:
Note: `json` contains the mongo orchestration config and should have been included long ago.